supplementary/alica_ros1/alica_tests/src/test/test_success_spam.cpp

```diff
@@ -1,49 +1,36 @@
-#include "test_alica.h"
-
-#include <alica_tests/BehaviourCreator.h>
-#include <alica_tests/ConditionCreator.h>
-#include <alica_tests/ConstraintCreator.h>
-#include <alica_tests/PlanCreator.h>
-#include <alica_tests/UtilityFunctionCreator.h>
-
+#include <alica/test/TestContext.h>
 #include <alica/test/Util.h>
-#include <engine/AlicaClock.h>
-#include <engine/AlicaEngine.h>
-#include <engine/Assignment.h>
-#include <engine/BasicBehaviour.h>
-#include <engine/DefaultUtilityFunction.h>
-#include <engine/IAlicaCommunication.h>
-#include <engine/PlanBase.h>
-#include <engine/PlanRepository.h>
-#include <engine/TeamObserver.h>
-#include <engine/model/Behaviour.h>
-#include <engine/model/Plan.h>
-#include <engine/model/RuntimeCondition.h>
-#include <engine/model/State.h>
-
+#include <alica_tests/Behaviour/Attack.h>
+#include <alica_tests/SimpleTestPlan1412252439925.h>
+#include <alica_tests/TestFixture.h>
 #include <gtest/gtest.h>
 
-namespace alica
-{
-namespace
-{
-
-class AlicaSpamSuccess : public AlicaTestFixture
+namespace alica::test
 {
-protected:
-    const char* getRoleSetName() const override { return "Roleset"; }
-    const char* getMasterPlanName() const override { return "BehaviorSuccessSpamMaster"; }
-};
 
 /**
  * Tests whether it is possible to run a behaviour in a primitive plan.
  */
-TEST_F(AlicaSpamSuccess, runBehaviour)
+TEST_F(TestFixture, runBehaviour)
```

Comment:
Ideally the plan corresponding to a test case should be named after the test case. In this example, the test is named runBehaviour & so I would expect the plan that it uses to be named RunBehaviourPlan (but I don't see any such plan) unless there is a specific need to reuse the same plan for multiple tests in which case you can make that clear with a comment